.github/workflows/build_package.yml

```diff
@@ -52,12 +52,12 @@ jobs:
             build-package: py-tf-compiler-tools-pkg
             experimental: false
 
-          # Macos packages.
-          - runs-on: macos-11
+          # MacOS packages.
+          - runs-on: [self-hosted, os-family=macOS, runner-group=releaser]
```

Comment:
We might want to use a pattern like https://github.com/openxla/stablehlo/pull/1052 for self hosted runners

```
    runs-on: ${{ github.repository == 'openxla/iree' && '${SELF_HOSTED}' || '${PUBLIC}'  }}
```

Reply:
Yeah I've been thinking about this. I would love to make it possible for a downstream to use their own self-hosted runners also. I think the best way to do this is to use repository variables: https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
Ha, good point. Switched to do this!